mailer.rst

```diff
@@ -551,7 +551,7 @@ and headers.
         framework:
             mailer:
                 envelope:
-                    sender: 'fabien@example.com'
+                    sender: 'Fabien <fabien@example.com>'
```

Comment:
What about documenting both with one in a comment?

Reply:
Do you mean

``` yaml
        framework:
            mailer:
                envelope:
                    sender: 'fabien@example.com'
                    # or sender: 'Fabien <fabien@example.com>'
```

or 

``` yaml
        framework:
            mailer:
                envelope:
                    sender: 'fabien@example.com'
                    # or with a display name: 'Fabien <fabien@example.com>'

```

Reply:
Yes for example
there are some config code in the documentation that show multiple way, could be great here also :)

Reply:
While merging we did this change. See https://github.com/symfony/symfony-docs/commit/f217868c4f32555ff6d26da0cfa01fa7d35566ea